modules/picard/sortvcf/main.nf

```diff
@@ -0,0 +1,34 @@
+process PICARD_SORTVCF {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::picard=2.26.10" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/picard:2.26.10--hdfd78af_0' :
+        'quay.io/biocontainers/picard:2.26.10--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("*_sorted.vcf"), emit: vcf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    picard \\
+        SortVcf \\
```

Comment:
does sortvcf need any setting for mem as in other picard tools ? ([Example](https://github.com/nf-core/modules/blob/fba1b381f5629d2dfce3ec2350dc03821fd275ad/modules/picard/mergesamfiles/main.nf#L33))

Reply:
Good point! I have added that option now.